python311Packages.dask: 2023.12.0 -> 2024.1.1

GaetanLepage opened the pull request:
## Description of changes

- Changelog `dask`: https://docs.dask.org/en/latest/changelog.html
- Changelog `dask-histogram`: https://github.com/dask-contrib/dask-histogram/releases/tag/2024.2.0
- Changelog `dask-awkward`: https://github.com/dask-contrib/dask-awkward/releases/tag/2024.2.0
- Changelog `awkward`: https://github.com/scikit-hep/awkward/releases/tag/v2.6.0

cc @veprbl @FRidh 

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage committed:
python311Packages.dask: 2023.12.0 -> 2024.1.1

Diff: https://github.com/dask/dask/compare/refs/tags/2023.12.0...2024.1.1

Changelog: https://docs.dask.org/en/latest/changelog.html

GaetanLepage committed:
python311Packages.dask-histogram: 2023.10.0 -> 2024.2.0

Diff: https://github.com/dask-contrib/dask-histogram/compare/refs/tags/2023.10.0...2024.2.0

Changelog: https://github.com/dask-contrib/dask-histogram/releases/tag/2024.2.0

GaetanLepage committed:
python311Packages.dask-awkward: 2024.1.2 -> 2024.2.0

Diff: https://github.com/dask-contrib/dask-awkward/compare/refs/tags/2024.1.2...2024.2.0

Changelog: https://github.com/dask-contrib/dask-awkward/releases/tag/2024.2.0

natsukium commented:
Result of `nixpkgs-review pr 285370` at 9f763367 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages marked as broken and skipped:</summary>
  <ul>
    <li>home-assistant-component-tests.dremel_3d_printer</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>python311Packages.caffeWithCuda</li>
    <li>python311Packages.caffeWithCuda.bin</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.tensorflow-datasets</li>
  </ul>
</details>
<details>
  <summary>20 packages failed to build:</summary>
  <ul>
    <li>deface</li>
    <li><a href="https://gist.github.com/natsukium/08805bcbf984eb5f7d87ac2f9a5e8183">napari</a> (python311Packages.napari)</li>
    <li><a href="https://gist.github.com/natsukium/6af565f1592b0104e02b7a5ff90608ff">python311Packages.aplpy</a></li>
    <li><a href="https://gist.github.com/natsukium/d40484d7a63525020b33bac2b3d830e8">python311Packages.clifford</a></li>
    <li>python311Packages.dalle-mini</li>
    <li><a href="https://gist.github.com/natsukium/a002a124482fa2cb382abc09aad431b8">python311Packages.dask-image</a></li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.insightface</li>
    <li><a href="https://gist.github.com/natsukium/7364a85d44d09ba0950f8de323336162">python311Packages.intake</a></li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.pvextractor</li>
    <li><a href="https://gist.github.com/natsukium/a73009e5b63baa184101de4545e13943">python311Packages.qcodes</a></li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li><a href="https://gist.github.com/natsukium/c6ae9245775655586500c027b9be80eb">python311Packages.sfepy</a></li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.textacy</li>
    <li>python311Packages.textnets</li>
    <li><a href="https://gist.github.com/natsukium/cbece4326c30983714d8d2c3255b2c90">python311Packages.torchrl</a></li>
    <li><a href="https://gist.github.com/natsukium/032d039f6b140b1c59821da2a8334107">rclip</a></li>
    <li>tts</li>
  </ul>
</details>
<details>
  <summary>70 packages built successfully:</summary>
  <ul>
    <li>csvkit</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>intensity-normalization (python311Packages.intensity-normalization)</li>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.caffe</li>
    <li>python311Packages.caffe.bin</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.fastai</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.niworkflows</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.spacy</li>
    <li>python311Packages.spacy-loggers</li>
    <li>python311Packages.spacy-lookups-data</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.wandb</li>
    <li>python311Packages.webdataset</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
  </ul>
</details>

GaetanLepage committed:
python311Packages.awkward-cpp: 28 -> 29

Changelog: https://github.com/scikit-hep/awkward/releases/tag/v29

GaetanLepage committed:
python311Packages.awkward: 2.5.2 -> 2.6.0

Diff: https://github.com/scikit-hep/awkward/compare/refs/tags/v2.5.2...v2.6.0

Changelog: https://github.com/scikit-hep/awkward/releases/tag/v2.6.0

veprbl commented:
Result of `nixpkgs-review pr 285370` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>14 packages marked as broken and skipped:</summary>
  <ul>
    <li>home-assistant-component-tests.dremel_3d_printer</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>python311Packages.caffeWithCuda</li>
    <li>python311Packages.caffeWithCuda.bin</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
  </ul>
</details>
<details>
  <summary>34 packages failed to build:</summary>
  <ul>
    <li>deface</li>
    <li>deface.dist</li>
    <li>napari (python311Packages.napari)</li>
    <li>napari.dist (python311Packages.napari.dist)</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.textacy</li>
    <li>python311Packages.textacy.dist</li>
    <li>python311Packages.textnets</li>
    <li>python311Packages.textnets.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>140 packages built:</summary>
  <ul>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>intensity-normalization (python311Packages.intensity-normalization)</li>
    <li>intensity-normalization.dist (python311Packages.intensity-normalization.dist)</li>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python311Packages.ome-zarr.dist)</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.caffe</li>
    <li>python311Packages.caffe.bin</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.fastai</li>
    <li>python311Packages.fastai.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyrender.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.spacy</li>
    <li>python311Packages.spacy-loggers</li>
    <li>python311Packages.spacy-loggers.dist</li>
    <li>python311Packages.spacy-lookups-data</li>
    <li>python311Packages.spacy-lookups-data.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.spacy.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.torchrl</li>
    <li>python311Packages.torchrl.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.wandb</li>
    <li>python311Packages.wandb.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
  </ul>
</details>

veprbl commented:
Result of `nixpkgs-review pr 285370` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>16 packages marked as broken and skipped:</summary>
  <ul>
    <li>home-assistant-component-tests.dremel_3d_printer</li>
    <li>home-assistant-component-tests.environment_canada</li>
    <li>python311Packages.caffeWithCuda</li>
    <li>python311Packages.caffeWithCuda.bin</li>
    <li>python311Packages.coffea</li>
    <li>python311Packages.coffea.dist</li>
    <li>python311Packages.correctionlib</li>
    <li>python311Packages.correctionlib.dist</li>
    <li>python311Packages.dm-sonnet</li>
    <li>python311Packages.dm-sonnet.dist</li>
    <li>python311Packages.stytra</li>
    <li>python311Packages.stytra.dist</li>
    <li>python311Packages.sunpy</li>
    <li>python311Packages.sunpy.dist</li>
    <li>python311Packages.tensorflow-datasets</li>
    <li>python311Packages.tensorflow-datasets.dist</li>
  </ul>
</details>
<details>
  <summary>36 packages failed to build:</summary>
  <ul>
    <li>deface</li>
    <li>deface.dist</li>
    <li>napari (python311Packages.napari)</li>
    <li>napari.dist (python311Packages.napari.dist)</li>
    <li>python311Packages.aplpy</li>
    <li>python311Packages.aplpy.dist</li>
    <li>python311Packages.clifford</li>
    <li>python311Packages.clifford.dist</li>
    <li>python311Packages.dalle-mini</li>
    <li>python311Packages.dalle-mini.dist</li>
    <li>python311Packages.dask-image</li>
    <li>python311Packages.dask-image.dist</li>
    <li>python311Packages.glueviz</li>
    <li>python311Packages.glueviz.dist</li>
    <li>python311Packages.insightface</li>
    <li>python311Packages.insightface.dist</li>
    <li>python311Packages.intake</li>
    <li>python311Packages.intake.dist</li>
    <li>python311Packages.mmcv</li>
    <li>python311Packages.mmcv.dist</li>
    <li>python311Packages.pvextractor</li>
    <li>python311Packages.pvextractor.dist</li>
    <li>python311Packages.qcodes-contrib-drivers</li>
    <li>python311Packages.qcodes-contrib-drivers.dist</li>
    <li>python311Packages.spectral-cube</li>
    <li>python311Packages.spectral-cube.dist</li>
    <li>python311Packages.textacy</li>
    <li>python311Packages.textacy.dist</li>
    <li>python311Packages.textnets</li>
    <li>python311Packages.textnets.dist</li>
    <li>python311Packages.vector</li>
    <li>python311Packages.vector.dist</li>
    <li>rclip</li>
    <li>rclip.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
  </ul>
</details>
<details>
  <summary>150 packages built:</summary>
  <ul>
    <li>csvkit</li>
    <li>csvkit.dist</li>
    <li>easyocr (python311Packages.easyocr)</li>
    <li>easyocr.dist (python311Packages.easyocr.dist)</li>
    <li>hydrus</li>
    <li>hydrus.doc</li>
    <li>intensity-normalization (python311Packages.intensity-normalization)</li>
    <li>intensity-normalization.dist (python311Packages.intensity-normalization.dist)</li>
    <li>ome_zarr (python311Packages.ome-zarr)</li>
    <li>ome_zarr.dist (python311Packages.ome-zarr.dist)</li>
    <li>python311Packages.agate-sql</li>
    <li>python311Packages.agate-sql.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.awkward</li>
    <li>python311Packages.awkward-cpp</li>
    <li>python311Packages.awkward-cpp.dist</li>
    <li>python311Packages.awkward.dist</li>
    <li>python311Packages.batchgenerators</li>
    <li>python311Packages.batchgenerators.dist</li>
    <li>python311Packages.bsuite</li>
    <li>python311Packages.bsuite.dist</li>
    <li>python311Packages.caffe</li>
    <li>python311Packages.caffe.bin</li>
    <li>python311Packages.casa-formats-io</li>
    <li>python311Packages.casa-formats-io.dist</li>
    <li>python311Packages.cf-xarray</li>
    <li>python311Packages.cf-xarray.dist</li>
    <li>python311Packages.crate</li>
    <li>python311Packages.crate.dist</li>
    <li>python311Packages.dask</li>
    <li>python311Packages.dask-awkward</li>
    <li>python311Packages.dask-awkward.dist</li>
    <li>python311Packages.dask-gateway</li>
    <li>python311Packages.dask-gateway.dist</li>
    <li>python311Packages.dask-glm</li>
    <li>python311Packages.dask-glm.dist</li>
    <li>python311Packages.dask-histogram</li>
    <li>python311Packages.dask-histogram.dist</li>
    <li>python311Packages.dask-jobqueue</li>
    <li>python311Packages.dask-jobqueue.dist</li>
    <li>python311Packages.dask-ml</li>
    <li>python311Packages.dask-ml.dist</li>
    <li>python311Packages.dask-mpi</li>
    <li>python311Packages.dask-mpi.dist</li>
    <li>python311Packages.dask-yarn</li>
    <li>python311Packages.dask-yarn.dist</li>
    <li>python311Packages.dask.dist</li>
    <li>python311Packages.datashader</li>
    <li>python311Packages.datashader.dist</li>
    <li>python311Packages.devito</li>
    <li>python311Packages.devito.dist</li>
    <li>python311Packages.distributed</li>
    <li>python311Packages.distributed.dist</li>
    <li>python311Packages.dremel3dpy</li>
    <li>python311Packages.dremel3dpy.dist</li>
    <li>python311Packages.env-canada</li>
    <li>python311Packages.env-canada.dist</li>
    <li>python311Packages.fastai</li>
    <li>python311Packages.fastai.dist</li>
    <li>python311Packages.glymur</li>
    <li>python311Packages.glymur.dist</li>
    <li>python311Packages.gradio</li>
    <li>python311Packages.gradio.dist</li>
    <li>python311Packages.gymnasium</li>
    <li>python311Packages.gymnasium.dist</li>
    <li>python311Packages.handout</li>
    <li>python311Packages.handout.dist</li>
    <li>python311Packages.imagecorruptions</li>
    <li>python311Packages.imagecorruptions.dist</li>
    <li>python311Packages.imageio</li>
    <li>python311Packages.imageio.dist</li>
    <li>python311Packages.intake-parquet</li>
    <li>python311Packages.intake-parquet.dist</li>
    <li>python311Packages.k-diffusion</li>
    <li>python311Packages.k-diffusion.dist</li>
    <li>python311Packages.lime</li>
    <li>python311Packages.lime.dist</li>
    <li>python311Packages.moviepy</li>
    <li>python311Packages.moviepy.dist</li>
    <li>python311Packages.mplhep</li>
    <li>python311Packages.mplhep.dist</li>
    <li>python311Packages.mung</li>
    <li>python311Packages.mung.dist</li>
    <li>python311Packages.muscima</li>
    <li>python311Packages.muscima.dist</li>
    <li>python311Packages.napari-console</li>
    <li>python311Packages.napari-console.dist</li>
    <li>python311Packages.napari-svg</li>
    <li>python311Packages.napari-svg.dist</li>
    <li>python311Packages.niworkflows</li>
    <li>python311Packages.niworkflows.dist</li>
    <li>python311Packages.omrdatasettools</li>
    <li>python311Packages.omrdatasettools.dist</li>
    <li>python311Packages.open-clip-torch</li>
    <li>python311Packages.open-clip-torch.dist</li>
    <li>python311Packages.pims</li>
    <li>python311Packages.pims.dist</li>
    <li>python311Packages.psd-tools</li>
    <li>python311Packages.psd-tools.dist</li>
    <li>python311Packages.pyfftw</li>
    <li>python311Packages.pyfftw.dist</li>
    <li>python311Packages.pyrender</li>
    <li>python311Packages.pyrender.dist</li>
    <li>python311Packages.pyvista</li>
    <li>python311Packages.pyvista.dist</li>
    <li>python311Packages.qcodes</li>
    <li>python311Packages.qcodes.dist</li>
    <li>python311Packages.reproject</li>
    <li>python311Packages.reproject.dist</li>
    <li>python311Packages.scikit-image</li>
    <li>python311Packages.scikit-image.dist</li>
    <li>python311Packages.sfepy</li>
    <li>python311Packages.sfepy.dist</li>
    <li>python311Packages.skrl</li>
    <li>python311Packages.skrl.dist</li>
    <li>python311Packages.slicedimage</li>
    <li>python311Packages.slicedimage.dist</li>
    <li>python311Packages.spacy</li>
    <li>python311Packages.spacy-loggers</li>
    <li>python311Packages.spacy-loggers.dist</li>
    <li>python311Packages.spacy-lookups-data</li>
    <li>python311Packages.spacy-lookups-data.dist</li>
    <li>python311Packages.spacy-transformers</li>
    <li>python311Packages.spacy-transformers.dist</li>
    <li>python311Packages.spacy.dist</li>
    <li>python311Packages.sparse</li>
    <li>python311Packages.sparse.dist</li>
    <li>python311Packages.streamz</li>
    <li>python311Packages.streamz.dist</li>
    <li>python311Packages.stumpy</li>
    <li>python311Packages.stumpy.dist</li>
    <li>python311Packages.test-tube</li>
    <li>python311Packages.test-tube.dist</li>
    <li>python311Packages.tifffile</li>
    <li>python311Packages.tifffile.dist</li>
    <li>python311Packages.torchrl</li>
    <li>python311Packages.torchrl.dist</li>
    <li>python311Packages.tsfresh</li>
    <li>python311Packages.tsfresh.dist</li>
    <li>python311Packages.uproot</li>
    <li>python311Packages.uproot.dist</li>
    <li>python311Packages.wandb</li>
    <li>python311Packages.wandb.dist</li>
    <li>python311Packages.webdataset</li>
    <li>python311Packages.webdataset.dist</li>
    <li>tartube</li>
    <li>tartube-yt-dlp</li>
    <li>tartube-yt-dlp.dist</li>
    <li>tartube.dist</li>
  </ul>
</details>
